tern-ontology/conceptual-information-model.md

```diff
@@ -248,9 +243,9 @@ Studies on the collected data provide us with an understanding of why and how th
 
 ## Spatial geometries
 
-Sites, site visits, samplings and observations can use the TERN Location Alignment vocabulary, which is a minimal ontology to tie together the GeoSPARQL vocabulary and the W3C Basic Geo WGS84 vocabulary. 
-TERN Location Alignment also defines a specialised Point class with properties to express the depth and elevation. 
-LineString and Polygon are also specialised to denote that a WKT literal is required.
+Sites, site visits, samplings and observations can use the TERN Location Alignment vocabulary, which is a minimal ontology to tie together the GeoSPARQL vocabulary and the W3C Basic Geo WGS84 vocabulary. See https://w3id.org/tern/ontologies/loc.
```

Comment:
My general recommendation is to use GeoSPARQL and WKT as much as possible. LOC gives the alternative of addresses or locations denoted by some text string, but in practice I think we will usually have coordinate-based geometries?

Reply:
> LOC gives the alternative of addresses or locations denoted by some text string

[`tern-loc:Point`](https://w3id.org/tern/ontologies/loc/Point) basically defines a few constraints to require `wgs:lat`, `wgs:long` and `geosparql:asWKT`. 

Not sure if we are looking at the same thing. I don't see any references to addresses or locations denoted by text string.

[These](https://linkeddata-dev.tern.org.au/viewers/tern-loc-ontology) are the current geometries defined in `tern-loc` for use with the TERN Ontology.

> but in practice I think we will usually have coordinate-based geometries?

Currently the 3 geometries defined in `tern-loc` all require a `wktLiteral` to represent the geometry.

Reply:
I was incorrect. I had not looked property at tern-loc - I had conflated it with https://www.w3.org/ns/locn